Zend/zend_vm_def.h

```diff
@@ -9380,6 +9380,130 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(203, ZEND_INIT_OBJ, VAR|UNUSED|THIS|CV, CONST, CACHE_SLOT, SPEC(OP_DATA=CONST|TMP|VAR|CV))
```

Comment:
```c
			property_object_node.op_type = IS_VAR;
			GET_NODE(&property_object_node, opline->result);

			znode property_name_node;
			property_name_node.op_type = IS_CONST;
			ZVAL_STR_COPY(&property_name_node.u.constant, property_name);
			opline = zend_emit_op(result, ZEND_INIT_OBJ, &property_object_node, &property_name_node);
```

1. ZEND_INIT_OBJ seems easy to misread as initializing a new object. ZEND_CLONE_ADD_PROP or something along those lines seems more self-explanatory
2. https://www.npopov.com/2017/04/14/PHP-7-Virtual-machine.html

    I would have thought OP1_TYPE would always be VAR? `$var` is CV, `$this` is THIS, but `clone $var` is VAR and `clone $this` is VAR (this operation acts on the clone, not the original?)? Can it be changed to `ZEND_INIT_OBJ, VAR,` without generating function definitions for those seemingly unreachable definitions - what happens as a result in the tests?

Reply:
(sorry for the force push, I've just added the --extensions-- section for the opcache test). I'll look into your comment(s) later! Thank you!

Reply:
1. What about `ZEND_CLONE_ASSIGN_PROP` in order to make it more clear that this opcode originates from `ZEND_ASSIGN_OBJ`?

Reply:
I prefer `ZEND_CLONE_INIT_PROP`